src/main/java/racingcar/domain/RacingResult.java

```diff
@@ -0,0 +1,33 @@
+package racingcar.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.dto.CarStatusDto;
+import racingcar.enums.Separator;
+
+public class RacingResult {
+    private List<CarStatusDto> carStatuses;
+
+    public RacingResult(List<CarStatusDto> carStatuses) {
+        this.carStatuses = carStatuses;
+    }
+
+    public String findWinner() {
+        return carStatuses.stream()
+                .filter(car -> car.position() == getMaxPosition())
+                .map(CarStatusDto::name)
+                .collect(Collectors.joining(Separator.JOIN_SEPARATOR.toString()));
+    }
+
+    private int getMaxPosition() {
+        return carStatuses.stream()
+                .mapToInt(CarStatusDto::position)
+                .max()
+                .getAsInt();
+    }
```

Comment:
`getMaxposition`이 반복적으로 호출되는 것 같습니다 :)
```suggestion
    public String findWinner() {
        final var maxPosition = getMaxPosition();
        return carStatuses.stream()
                .filter(car -> car.position() == maxPosition)
                .map(CarStatusDto::name)
                .collect(Collectors.joining(Separator.JOIN_SEPARATOR.toString()));
    }

    private int getMaxPosition() {
        return carStatuses.stream()
                .mapToInt(CarStatusDto::position)
                .max()
                .getAsInt();
    }
```

Reply:
getMaxPosition()으로 구한 값은 한번만 구해서 사용해도 되는데
반복을 없앨 수 있는 방법이 있었는지 몰랐습니다!
말씀해주신 var라는 타입에 대해 한번 알아봐야겠네요 감사합니다!